.gitignore

```diff
@@ -1,5 +1,7 @@
 .env
 .tmp-id
+.idea
+.vscode
```

Comment:
You can move those to your local `.git/info/exclude` to avoid adding editor specific files here.